public/livesearch-autoparco-tipologie.php

```diff
@@ -0,0 +1,55 @@
+<?php
+    session_start();
+
+    if ($_SESSION['auth'] < 1 ) {
+        header("Location: index.php?s=1");
+        exit();
+    }
+
+    include '../db-connessione-include.php';
+    include 'maledetti-apici-centro-include.php'; //ATTIVA O DISATTIVA IL MAGIC QUOTE PER GLI APICI
+
+    function __autoload ($class_name) { //funzione predefinita che si occupa di caricare dinamicamente tutti gli oggetti esterni quando vengono richiamati
```

Comment:
Fai `git pull origin master` per tirarti le ultime modifiche che sono già in master, e rimuovi poi del tutto la funzione `__autoload` (righe 12-14), perché non è più necessaria e rompe tutto il nuovo codice, come @biagiosaitta ha già sperimentato (AWS).

Reply:
Tieni presente che il file `db-connessione-include.php` si chiama ora `core.php` (perché non contiene solamente roba relativa al database). Dopo aver tirato le ultime modifiche, dovrai cambiare tutte le occorrenze da

```
include '../db-connessione-include.php';
```

a

```
include '../core.php';
```